src/ImageRunModal.jsx

```diff
@@ -119,6 +119,7 @@ const Volume = ({ id, item, onChange, idx, removeitem, additem }) =>
                        placeholder={_("Container path")}
                        value={item.containerPath || ''}
                        onChange={e => onChange(idx, 'containerPath', e.target.value)} />
+                <div />
```

Comment:
This div either should not be here or has wrong indentation

Reply:
It is on the same level as other components. I am afraid I do not follow.
It is a small hack, as I needed some empty element to make things wrap into the next row.

Reply:
This </div> line that you added just closes the line  <div role='group' className='ct-form-split'> on 112.
If that's done on purpose you should remove 4 spaces because the line that it closes is 4 spaces behind the one you added :)

Reply:
Ups I just saw it's an empty div. Overlooked that as an closing element.

Reply:
No, this is `<div />` which is short form for `<div></div>` and not `</div>` :)